src/main/resources/jenkins/plugins/purgejobhistory/PurgeJobHistoryAction/index.properties

```diff
@@ -0,0 +1,2 @@
+title=Purge History {0}
```

Comment:
Maybe too verbose for the sidebar. The full explanation is available when you click the link.